src/figuresClasses.ts

```diff
@@ -2,10 +1,75 @@
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const sides = [a, b, c].sort((x, y) => x - y);
+
+    if (sides[2] >= sides[0] + sides[1]) {
+      const message =
+        'The sum of any two sides must be greater than the third side.';
+
+      throw new Error(message);
```

Comment:
[CODE STYLE] When throwing an error, try to provide a detailed description in the error message. In this case, the error message could include information about which sides are causing the problem.